### PR TITLE
[skip ci] Bash helpers for managing VCHs

### PR DESCRIPTION
**Pull request by hickeng**
A set of bash functions for:
deploy
delete
ls
insecpt
ssh

These make use of environment variables, which are set via bash functions
sourced from ~/.vic

### CODE REVIEW

**Comment by hickeng** [comment]
@caglar10ur you asked for my helper script

File: infra/scripts/bash-helpers.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	if [ -z ${DOCKER_TLS_VERIFY+x} ]; then
+		alias docker='docker --tls'
+	fi
+
+        popd
+
+}
+
+vic-rm () {
+        ~/vic/bin/vic-machine-linux delete -target="$GOVC_URL" -compute-resource="$COMPUTE" --name=${VIC_NAME:-ghicken-test} --force $*
```
Comment (fdawg4l) [inline infra/scripts/bash-helpers.sh]: `-ghicken-test`

File: infra/scripts/bash-helpers.sh [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	~/vic/bin/vic-machine-linux ls -target="$GOVC_URL" $*
+}
+
+vic-ssh () {
+	unset keyarg
+	if [ -e $HOME/.ssh/authorized_keys ]; then
+		keyarg="--authorized_keys=$HOME/.ssh/authorized_keys"
+	fi
+
+        out=$(~/vic/bin/vic-machine-linux debug -target="$GOVC_URL" -compute-resource="$COMPUTE" --name=${VIC_NAME:-ghicken-test} --enable-ssh $keyarg --rootpw=password $*)
```
Comment (fdawg4l) [inline infra/scripts/bash-helpers.sh]: More `-ghicken-test`.

File: infra/scripts/bash-helpers.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        popd
+
+}
+
+vic-rm () {
+        ~/vic/bin/vic-machine-linux delete -target="$GOVC_URL" -compute-resource="$COMPUTE" --name=${VIC_NAME:-ghicken-test} --force $*
+}
+
+vic-inspect () {
+       ~/vic/bin/vic-machine-linux inspect -target="$GOVC_URL" -compute-resource="$COMPUTE" --name=${VIC_NAME:-ghicken-test} $*
```
Comment (fdawg4l) [inline infra/scripts/bash-helpers.sh]: Samesees.

File: infra/scripts/bash-helpers.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+vic-ls () {
+	~/vic/bin/vic-machine-linux ls -target="$GOVC_URL" $*
```
Comment (fdawg4l) [inline infra/scripts/bash-helpers.sh]: How about `$GOPATH/bin/` intead of `~/vic/bin`?

**Comment by caglar10ur** [comment]
<insert a mind blow gif>

LGTM

File: infra/scripts/bash-helpers.sh (hunk 4 above)
Comment (hickeng) [inline infra/scripts/bash-helpers.sh]: done all of them I hope

**Comment by caglar10ur** [comment]
still lgtm

**Comment by hickeng** [comment]
poke

**Comment by hickeng** [comment]
poke

**Comment by hickeng** [comment]
poke

**Comment by hickeng** [comment]
WTF

**Comment by anchal-agrawal** [comment]
LGTM

**Pull request merged**